docs/source/algorithms/CreatePoleFigureTableWorkspace-v1.rst

```diff
@@ -0,0 +1,92 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Create a table with the information required to produce a pole figure, namely, a collection of alphas, betas,
```

Comment:
Maybe some description of what `alpha` and `beta` correspond to would be nice here?